funsor/delta.py

```diff
@@ -118,6 +126,49 @@ def eager_integrate(delta, integrand, reduced_vars):
     return Integrate(log_measure, integrand, reduced_vars)
 
 
+def solve(expr, value):
+    """
+    Tries to solve for free inputs of an ``expr`` such that ``expr == value``,
+    and computes the log-abs-det-Jacobian of the resulting substitution.
```

Comment:
If we merge #78 I think we should be able to factor this into a unification step and a log-abs-det-Jacobian step, where unification is handled by `unification.unify`.

Reply:
Sounds like a good follow-up PR.